src/main/java/teammates/common/datatransfer/questions/FeedbackTextQuestionDetails.java

```diff
@@ -72,7 +72,7 @@ public String getQuestionWithoutExistingResponseSubmissionFormHtml(
             boolean sessionIsOpen, int qnIdx, int responseIdx, String courseId, int totalNumRecipients) {
         return Templates.populateTemplate(
                 FormTemplates.TEXT_SUBMISSION_FORM,
-                Slots.DISABLED, sessionIsOpen ? "" : "disabled",
+                Slots.IS_SESSION_CLOSED, sessionIsOpen ? "false" : "true",
```

Comment:
@HirdayGupta I did change the line to this
```javascript
Slots.IS_SESSION_CLOSED, !sessionIsOpen ? "true" : "false",
```
But 2 PMD test failed to suggest me to avoid using `!`